public_html/profiles/social/themes/socialbase/components/contrib/bootstrap/_utilities.scss

```diff
@@ -52,5 +52,4 @@
 
 .affix {
   position: fixed;
-  top: $navbar-height;
```

Comment:
This line should not be there indeed. I compared it with https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_utilities.scss.